docs/pressable.md

```diff
@@ -0,0 +1,175 @@
+---
+id: pressable
+title: Pressable
+---
+
+Pressable is a primitive core component wrapper that is used to detect various types of interaction states on any of it's defined children.
+
+```jsx
+<Pressable onPress={onPressFunction}>
+  <Text>I'm pressable!</Text>
+</Pressable>
+```
+
+## Example
+
+```
+import * as React from 'react';
+import {
+  Pressable,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+
+export default function App() {
+  const [timesPressed, setTimesPressed] = useState(0);
+
+  let textLog = '';
+  if (timesPressed > 1) {
+    textLog = timesPressed + 'x onPress';
+  } else if (timesPressed > 0) {
+    textLog = 'onPress';
+  }
+
+  return (
+    <View>
+      <Pressable
+          onPress={() => {
+            setTimesPressed(current => current + 1);
+          }}>
+          {({pressed}) => (
+            <Text style={styles.text}>{pressed ? 'Pressed!' : 'Press Me'}</Text>
+          )}
+        </Pressable>
+      </View>
+      <View style={styles.logBox}>
+        <Text testID="pressable_press_console">{textLog}</Text>
+      </View>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  text: {
+    fontSize: 16,
+   },
+   logBox: {
+    padding: 20,
+    margin: 10,
+    borderWidth: StyleSheet.hairlineWidth,
+    borderColor: '#f0f0f0',
+    backgroundColor: '#f9f9f9',
+  },
+});
+```
+
+---
+
+# Reference
+
+## Props
+
+### `android_disableSound`
+
+If true, doesn't play Android system sound on press.
+
+| Type    | Required | Platform |
+| ------- | -------- | -------- |
+| boolean | No       | Android  |
+
+### `android_rippleColor`
+
+Enables the Android ripple effect and configures its color.
+
+| Type       | Required | Platform |
+| ---------- | -------- | -------- |
+| ColorValue | No       | Android  |
+
+### `children`
+
+Either children or a render prop that receives a boolean reflecting whether the component is currently pressed.

```

Comment:
Maybe instead of "render prop" it should say "function"

Reply:
Also, it receives an object with the press state, of which `pressed` is a boolean. Right?